### DISCUSSION TITLE
Performance issues on 5.0.3 with open devtools?

### DISCUSSION THREAD

**Discussion by otakueadwine** [clauderic/dnd-kit#700] [Q&A]
Hi guys, I just upgraded to the new version, 5.0.3 from using 4.0.3.
So far fixed all the breaking changes, nothing is throwing any errors in my console.

The only noticeable thing is a major performance downgrade when I have my devtools open.
I was wondering if anyone else is encountering this as well with the new version, as I'm not quite sure if I'm overlooking anything in our own code or not? With closed devtools the performance seems to be allright. (The grid has about 130 items on it btw)

5.0.3 with open devtools:
https://user-images.githubusercontent.com/8110976/163371064-929231ec-66c1-4bde-aabf-680793d74311.mov

4.0.3 with open devtools:
https://user-images.githubusercontent.com/8110976/163371824-d75baa6d-be79-40c5-a438-ba9b3a2c870a.mov

**Comment by sebastianservat** [comment]
@otakueadwine If you share a CodeSandbox with your 130 items on grid, happy to test it on my side.
